stackdriver_trace.c

```diff
@@ -243,6 +247,12 @@ static stackdriver_trace_span_t *stackdriver_trace_begin(zend_string *function_n
 
     span->start = stackdriver_trace_now();
     span->name = zend_string_copy(function_name);
+
+#if PHP_VERSION_ID < 70100
+    if (!BG(mt_rand_is_seeded)) {
+        php_mt_srand(GENERATE_SEED());
```

Comment:
in PHP 7.0.x, the random number generator must be seeded before calling `php_mt_rand()`